sdk/appconfiguration/azure-appconfiguration-provider/setup.py

```diff
@@ -75,5 +75,6 @@
         "azure-core>=1.28.0",
         "azure-appconfiguration>=1.6.0",
         "azure-keyvault-secrets>=4.3.0",
+        "dnspython>=2.6.1",
```

Comment:
Since we are taking a dependency on an external library, we should add third party license notice for dnspython ([from here](https://github.com/rthalley/dnspython/blob/main/LICENSE)). 

Here's a sample notice in the python SDK: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/monitor/azure-monitor-opentelemetry/NOTICE.txt

And an example from .NET provider: https://github.com/Azure/AppConfiguration-DotnetProvider/blob/96a5e5a07b8d4b4aa55d31bcab14219ee6912a04/NOTICE#L21

Reply:
We don't need it as it isn't included in our release bundle.